packages/virtualized-lists/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@react-native-mac/virtualized-lists",
```

Comment:
```suggestion
  "name": "@react-native-macos/virtualized-lists",
```

Why "mac"?

Reply:
Somebody else is squatting on the @react-native-macos scope on NPMJS. We're filing a dispute for it, but we don't have an estimate for when it'll be resolved.

Reply:
What? Who?

Reply:
Talked offline. Let's please make sure we track this.. I would want that namespace long term.

Reply:
Will do.